pages/api/upload/delete.js

```diff
@@ -0,0 +1,27 @@
+import cloudinary from "cloudinary";
+
+cloudinary.config({
+  cloud_name: process.env.CLOUDINARY_CLOUD_NAME,
+  api_key: process.env.CLOUDINARY_API_KEY,
+  api_secret: process.env.CLOUDINARY_SECRET,
+});
+
+export default async function handler(req, res) {
+  if (req.method === "DELETE") {
```

Comment:
please don't use abbreviations like req or res.

Reply:
Thank you! I'll change that!